core/src/scene/stops.cpp

```diff
@@ -181,6 +181,29 @@ auto Stops::Widths(const YAML::Node& _node, const MapProjection& _projection, co
     return stops;
 }
 
+auto Stops::Numbers(const YAML::Node& node) -> Stops {
+    Stops stops;
+    if (!node.IsSequence()) { return stops; }
+
+    float lastKey = 0;
+
+    for (const auto frameNode : node) {
+        if (!frameNode.IsSequence() || frameNode.size() != 2) { continue; }
+
+        float key = frameNode[0].as<float>();
```

Comment:
Yeah it's not terribly robust, it's just the way that every other `Stops` factory method is implemented. I intend to refactor this whole class in the not-too-distant future, so don't think of it as permanent :)